BraintreeCore/src/main/java/com/braintreepayments/api/PreferredPaymentMethodsResult.java

```diff
@@ -4,8 +4,9 @@
 import org.json.JSONObject;
 
 /**
- *  Contains information about which payment methods are preferred on the device.
- *  This class is currently in beta and may change in future releases.
+ * Contains information about which payment methods are preferred on the device.
+ * This class is currently in beta and may change in future releases.
```

Comment:
Wonder if this should be updated to "may change or be removed" since it seems like the latter is the path that we may go down

Reply:
Good call. I replaced "may change" with "may be removed."